pynndescent/rp_trees.py

```diff
@@ -441,6 +441,7 @@ def sparse_euclidean_random_projection_split(inds, indptr, data, indices, rng_st
     return indices_left, indices_right, hyperplane, hyperplane_offset
 
 
+@numba.jit()
```

Comment:
I removed this numba decorator (and equivalent ones for the following three functions) since it was producing warnings from numba, and wasn't actually being JITed (`euclidean_random_projection_split` is the real JIT target).

Reply:
I'm happy enough to see them get removed. I will try to get the jittable trees added soon.